.github/workflows/ci-windows-artifacts.yml

```diff
@@ -46,7 +43,18 @@ jobs:
     - name: Build libyuv
       if: steps.cache-ext.outputs.cache-hit != 'true'
       working-directory: ./ext
-      run: ./libyuv.cmd
+      # Use clang-cl to build libyuv. The assembly code in libyuv is written in the
+      # GCC inline assembly format. Visual C++ doesn't support GCC inline assembly.
+      run: >
+        set "CC=clang-cl"
+
+        set "CXX=clang-cl"
+
+        ./libyuv.cmd
+
+        set "CC="
+
+        set "CXX="
```

Comment:
In PowerShell, these two lines should read:

```
$Env:CC = ''

$Env:CXX = ''
```

We may be able to omit these if the environment variables affect only the current step.

Reply:
Thank you for the PowerShell syntax research.

> We may be able to omit these if the environment variables affect only the current step.

It works as is so I am keeping them.